lineage/states/StateDistributionGamma.py

```diff
@@ -190,3 +189,46 @@ def time_censor(cell, desired_experiment_time):
             # the daughters are no longer observed
             cell.left.observed = False
             cell.right.observed = False
+
+def atonce_estimator(x_list, gammas_list):
+    """ User-defined way of estimating the parameters given a list of the tuples of observations from a group of cells. """
+    # unzipping the list of tuples
+    x_data = []
+    for x in x_list:
+        x_data.append(np.array(x))
+
+    assert len(x_data) == len(x_list)
+    # getting the observations as individual lists
+    # {
+    bern_obs = [x[:, 0].astype('bool') for x in x_data]
```

Comment:
Bernoulli shouldn't be handled in here. We could add constraints on it, too, but not right now. It can be handled by calling the normal estimators. Then, the constrained gamma solution can overwrite the normal gamma estimator.

Reply:
removed the bernoulli estimation from the estimator (that is in another branch)